### ISSUE TITLE
LED Strip Lights constantly not responding in Home App and Homebridge Accessories Section. 

### ISSUE THREAD

**Issue by TevanieP** [bwp91/homebridge-govee#76]
### What issue do you have? Please be as thorough and explicit as possible.
When changing the brightness or color of Govee LED Light Strips (Model H6154), accessory will constantly not respond or be extremely slow to requests

### Details of your setup.
* Do you use (1) Homebridge (+ config-ui), (2) Homebridge (CLI) or (3) HOOBS? 
I use Homebridge. 

* Which version of Homebridge/HOOBS do you have?
I have version 1.3.4

* Which platform do you run Homebridge/HOOBS on (e.g. Raspberry Pi/Windows/HOOBS Box)? Please also mention your version of Node.js/NPM if known.

I run Homebridge on Raspberry Pi, Node.js : 14.6.1, NPM : 7.12.1

* Which version of this plugin (homebridge-govee) do you have? Has the issue started since upgrading from a previous version?

I have version 3.2.3, the issue has occurred in this version and the last. 

* Which Govee devices do you have that are causing issues? Please include product models if applicable.
Model H6154, RGB LED Light Strip

### Please paste any relevant logs below.

```
[10/05/2021, 22:58:12] [Govee] [Bedroom Strip LEDs] could not be updated as Request failed with status code 400 at createError (/usr/local/lib/node_modules/homebridge-govee/node_modules/axios/lib/core/createError.js:16:15).
```

**Comment by bwp91** [comment]
Hi @TevanieP 
Do you get this message in the log when you try to control brightness? colour? both?
i can understand brightness, but not colour.
you may find you get this message when you try to change the brightness to over 39% in homekit?

**Comment by bwp91** [comment]
Please try the beta I have just released (v3.2.4-beta.0) to see if it was the brightness causing the issue.

**Comment by TevanieP** [comment]
My apologies, sorry for not clarifying enough. I get that message when I change the brightness and I believe it happens for color as well. I’ll do more testing later on today and try to differentiate if its both or just brightness. 

And It doesn't only give me that error over “39%” it happens to all percentages of brightness.

**Comment by TevanieP** [comment]
> Please try the beta I have just released (v3.2.4-beta.0) to see if it was the brightness causing the issue.

I've updated the plugin and yes I'm still having the problem you mentioned. I can't place my brightness over 39%, If I do, I  receive that same error in the log. And it doesn't happen when I change the color, that was my miscalculation.

**Comment by bwp91** [comment]
This is strange as the v3.2.4-beta.0 should have fixed the 39% brightness issue.

Could you turn on debug logging and set the brightness of the device (via the home app) to 100% and paste the logs here of this 'process'?

Thanks
